crates/sui-types/src/messages_checkpoint.rs

```diff
@@ -111,7 +112,9 @@ impl AsRef<[u8; 32]> for CheckpointDigest {
     }
 }
 
-#[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
+#[derive(
+    Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize, JsonSchema,
+)]
```

Comment:
This will serialize to a byte array, we need to do customise the serde like TransactionDigest https://github.com/MystenLabs/sui/blob/2d9a057ad0c07a2c745b9e416948d5ec2e5a7363/crates/sui-types/src/base_types.rs#L287-L291